package/python-kiwi-spec-template

```diff
@@ -91,17 +91,26 @@ Recommends:     dpkg
 %endif
 %if 0%{?suse_version}
 Recommends:     gpg2
-Recommends:     dnf
 Recommends:     debootstrap
 Recommends:     dpkg
+%if 0%{?suse_version} >= 1650
+Recommends:     dnf
+%endif
 %endif
 # package managers required by distro
 %if 0%{?fedora} || 0%{?rhel} >= 8 || 0%{?suse_version} >= 1650
 Provides:       kiwi-packagemanager:microdnf
 Requires:       microdnf
 %endif
+%if 0%{?fedora} >= 39
+Requires:       dnf5
```

Comment:
Shouldn't we have `kiwi-packagemanager:dnf5` Provides too?

Reply:
yes good point, will fix it. Thanks

Reply:
done